cmd/src/lsif_upload.go

```diff
@@ -64,7 +65,11 @@ func handleLSIFUpload(args []string) error {
 		return handleLSIFUploadError(nil, err)
 	}
 
-	uploadID, err := upload.UploadIndex(lsifUploadFlags.file, lsifUploadOptions(out))
+	client := api.NewClient(api.ClientOpts{
+		Flags: lsifUploadFlags.apiFlags,
+	})
+
```

Comment:
It might be a good idea to log a warning if the client is initialized without TLS validation.

Reply:
Thats a good idea, I'll follow up with a separate PR for that, as those changes would be more wide-reaching across teams 🙂